examples/nextjs/src/graphql/codegen.example.yml

```diff
@@ -0,0 +1,11 @@
+schema:
```

Comment:
rename to codegen.yml and update the README since we can use env vars

Reply:
Done